src/sentry/static/sentry/app/components/stackedBarChart.tsx

```diff
@@ -11,7 +11,56 @@ import {use24Hours, getTimeFormat} from 'app/utils/dates';
 import theme from 'app/utils/theme';
 import {formatFloat} from 'app/utils/formatters';
 
-class StackedBarChart extends React.Component {
+type Point = {x: number; y: number[]; label?: string};
+type Points = Point[];
+type Series = Array<{
+  data: Array<{x: number; y: number}>;
+  label?: string;
+  color?: string;
+}>;
+type Marker = {
+  x: number;
+  label?: string;
+  className?: string;
```

Comment:
I believe that we usually place className in the last position of the type object :)

Reply:
moved!